firmware/inc/display.h

```diff
@@ -89,9 +89,9 @@ typedef enum time_formats_t
 
 typedef enum row_number_t
 {
-    ROW_1 = 1,
-    ROW_2 = 2,
-    ROW_3 = 3
+    ROW_1 = 0,
+    ROW_2 = 1,
+    ROW_3 = 2

```

Comment:
Any particular reason for this change?

Reply:
Just changed because they are used for indexing arrays in the driver. So need to start at 0